WorkforceDataApi/src/WorkforceDataApi/Program.cs

```diff
@@ -1,6 +1,145 @@
+using Hangfire;
+using Hangfire.PostgreSql;
+using Mapster;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.OpenApi.Models;
+using Sentry.AspNetCore;
+using Serilog;
+using WorkforceDataApi.Csv;
+using WorkforceDataApi.Models;
+using WorkforceDataApi.Responses;
+using WorkforceDataApi.Services;
+using WorkforceDataApi.Services.BackgroundJobs;
+
 var builder = WebApplication.CreateBuilder(args);
+builder.WebHost.UseKestrel(options =>
+{
+    options.AddServerHeader = false;
+});
+
+builder.Host.UseSerilog((ctx, config) =>
+    config.ReadFrom.Configuration(ctx.Configuration));
+
+builder.Services.AddApplicationInsightsTelemetry();
+
+if (builder.Environment.IsProduction())
+{
+    builder.WebHost.UseSentry();
+    builder.Services.Configure<SentryAspNetCoreOptions>(options =>
+    {
+        var hostingEnvironmentName = builder.Configuration["EnvironmentName"];
+        if (!string.IsNullOrEmpty(hostingEnvironmentName))
+        {
+            options.Environment = hostingEnvironmentName;
+        }
+
+        var gitSha = builder.Configuration["GitSha"];
+        if (!string.IsNullOrEmpty(gitSha))
+        {
+            options.Release = gitSha;
+        }
+    });
+}
+
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen(c =>
+{
+    c.SwaggerDoc("v1", new OpenApiInfo() { Title = "TRN Generation API", Version = "v1" });
+});
+builder.Services.AddDbContext<WorkforceDbContext>();
+
+if (builder.Environment.IsDevelopment())
+{
+    builder.Configuration
```

Comment:
Not sure we need this, I think it's added by default

Reply:
You are correct, user secrets are automatically added in the builder